WalletWasabi.Fluent/Models/Transactions/PrivacySuggestionsResult.cs

```diff
@@ -1,18 +1,76 @@
 using System.Collections.Generic;
-using System.Linq;
+using System.Threading.Tasks;
+using WalletWasabi.Fluent.Extensions;
 
 namespace WalletWasabi.Fluent.Models.Transactions;
 
-public record PrivacySuggestionsResult
+public record PrivacySuggestionsResult(IAsyncEnumerable<PrivacyItem> Items)
```

Comment:
What is the point of `Items`? It seems to me we only use the `_combinedResults`

Reply:
@soosr Removed. That was a leftover from a previous version of the code.